contributing.md

```diff
@@ -1,34 +1,93 @@
 # Contributing to FingerprintJS
 
-## Working with the code
+Thanks for taking the time to contribute!
+Here you can find ways to make FingerprintJS better, as well as tips and guidelines.
 
-Make sure you have [Yarn](https://yarnpkg.com) installed.
+This project and everyone participating in it is governed by the [Code of Conduct](code_of_conduct.md).
+By participating, you are expected to uphold this code.
 
-### Development playground
+## How you can contribute
+
+### Reporting an issue
+
+If you've noticed a bug, have an idea or a question,
+feel free to [create an issue](https://github.com/fingerprintjs/fingerprintjs/issues/new/choose) or [start a discussion](https://github.com/fingerprintjs/fingerprintjs/discussions/new/choose).
+
+Before you start, please [search](https://github.com/search?q=repo%3Afingerprintjs%2Ffingerprintjs&type=code) for your topic.
+There is a chance it has already been discussed.
+
+When you create an issue, the description is pre-filled with a template text.
+Please fill the missing information carefully, it will help us solve your issue faster.
+If you want share a piece of code or the library output with us, please wrap it in a ` ``` ` block and make sure you include all the information.
+
+### Creating a pull request
+
+If you want to fix a bug, add a source of entropy or make any other code contribution, please [create a pull request](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project).
+
+After you clone the repository, check the [Working with code](#working-with-code) section to learn how to run, check and build the code.
+
+In order for us to review and accept your code contributions, please follow these rules:
+- Your code quality should be at least as good as the code that you modify;
```

Comment:
Some list items in the document end with `.` and some end with `;`. Periods are [more](https://developers.google.com/style/lists) [conventional](https://learn.microsoft.com/en-us/style-guide/scannable-content/lists), I would unify it and end all list item sentences with a period.

```suggestion
- Your code quality should be at least as good as the code you modify.
```